core/src/test/scala/com/microsoft/azure/synapse/ml/core/test/fuzzing/Fuzzing.scala

```diff
@@ -409,6 +413,204 @@ trait PyTestFuzzing[S <: PipelineStage] extends TestBase with DataFrameEquality
     Files.write(
       FileUtilities.join(testDir, "test_" + camelToSnake(testClassName) + ".py").toPath,
       testClass.getBytes(StandardCharsets.UTF_8))
+    if (classOf[TestBase].isAssignableFrom(this.getClass)) {
+      try {
+        afterAll()
+      } catch {
+        case _: AbstractMethodError => {}
+      }
+    }
+  }
```

Comment:
can you help me understand why is this code necessary?

Reply:
No, I can't, 'cause I don't know either. Probably this was from when I was still getting used to traits, and I was confused, as usual. Deleted.